containers/local.env

```
@@ -8,6 +8,11 @@ FILES_DB_BUCKET=testbucket
 FILES_DB_ACCESS_KEY=minioadmin
 FILES_DB_SECRET_KEY=minioadmin
 
+STRIPE_SECRET=sk_test_51IespfBVvvKoCGaYoSWN8arbtnLKmYS9UupcF9xSqDtF5NBUf4l7pyZrceMQcj1wLTrw86Sh6GVMiDOXfAj90nvA002kUf7TjF
+STRIPE_SIGNING_SECRET=get value from stripe cli or dashboard
```

```suggestion
STRIPE_SIGNING_SECRET="" #get value from stripe cli or dashboard
```
The script fails for me trying to execute the text with spaces, so this just gives it an empty value and leaves the comment in there.